### PR TITLE
Dredd init fix

### PR DESCRIPTION
**Pull request by netmilk**
- disabled Apiary test reporter configuration
- fixed passing of additional arguments when dredd.yml present
- fixed README formatting

### CODE REVIEW

File: src/interactive-config.coffee [hunk 1]
```diff
@@ -40,7 +40,12 @@ interactiveConfig.prompt = (config = {}, callback) ->
     name: "apiaryApiKey"
     message: "Please enter Apiary API key or leave empty for anonymous reporter"
     default: config['custom']?['apiaryApiKey']
-    when: (answers) -> (answers['apiary'] == true && ! config['custom']?['apiaryApiKey']?)
+    when: (answers) ->
+      # it's not GA on the Apiary side yet
+      if process.env['APIARY_BETA']?
+        (answers['apiary'] == true && ! config['custom']?['apiaryApiKey']?)
```
Comment (kuba-kubula) [inline src/interactive-config.coffee]: @netmilk this `if` really bothers me. It's hard to read, really. :-1:

File: src/interactive-config.coffee [hunk 2]
```diff
@@ -48,10 +53,15 @@ interactiveConfig.prompt = (config = {}, callback) ->
...
-      (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
-      (answers['apiary'] == true && answers['apiaryApiKey'] != '')
-    )
+    when: (answers) ->
+      # it's not GA on the Apiary side yet
+      if process.env['APIARY_BETA']?
+        (
+          (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
+          (answers['apiary'] == true && answers['apiaryApiKey'] != '')
+        )
```
Comment (kuba-kubula) [inline src/interactive-config.coffee]: `answers.apiary is true && answers.apiaryApiKey && config.custom?.apiaryApiName` ?

File: src/interactive-config.coffee (hunk 1 above)
Comment (netmilk) [inline src/interactive-config.coffee]: @kuba-kubula you mean that condition statement or usage of `if`? I agree, but it was there before this PR :)

File: src/interactive-config.coffee (hunk 2 above)
Comment (netmilk) [inline src/interactive-config.coffee]: @kuba-kubula Sorry, I
 don't get what do you mean.

Comment (netmilk) [inline src/interactive-config.coffee]: I assume you propose this statement instead.
```
answers.apiary is true && answers.apiaryApiKey && config.custom?.apiaryApiName
```

**Pull request merged**